### PR TITLE
29 update user html view

### PR DESCRIPTION
**Pull request by gonciarstef**

### CODE REVIEW

File: src/main/resources/templates/user.html [hunk 1]
```diff
@@ -8,11 +8,79 @@
...
+                            <p class="text-muted mb-1">Headline</p>
+                            <p class="text-muted mb-4">Bay Area, San Francisco, CA</p>
+                            <div class="d-flex justify-content-center mb-2">
+                                <button type="button" class="btn btn-primary">Follow</button>
+                                <button type="button" class="btn btn-outline-primary ms-1">Message</button>
+                            </div>
+                        </div>
+                    </div>
 
-    <h1 class="text-danger">Under construction</h1>
```
Comment (Dariusionut) [inline src/main/resources/templates/user.html:15]: This line can pe removed

File: src/main/resources/templates/user.html [hunk 2]
```diff
@@ -8,11 +8,79 @@
...
+                                    <p class="text-muted mb-0">Johnatan Smith</p>
+                                </div>
+                            </div>
+                            <hr>
+                            <div class="row">
+                                <div class="col-sm-3">
+                                    <p class="mb-0">Email</p>
+                                </div>
+                                <div class="col-sm-9">
+                                    <p class="text-muted mb-0">example@example.com</p>
```
Comment (Dariusionut) [inline src/main/resources/templates/user.html:47]: th:text="${user.email}" as our users has an email property

File: src/main/resources/templates/user.html [hunk 3]
```diff
@@ -8,11 +8,79 @@
...
+                </div>
+                <div class="col-lg-8">
+                    <div class="card mb-4">
+                        <div class="card-body">
+                            <div class="row">
+                                <div class="col-sm-3">
+                                    <p class="mb-0">Full Name</p>
+                                </div>
+                                <div class="col-sm-9">
+                                    <p class="text-muted mb-0">Johnatan Smith</p>
```
Comment (Dariusionut) [inline src/main/resources/templates/user.html:38]: th:text="${user.firstName + user.lastName}"

File: src/main/resources/templates/user.html [hunk 4]
```diff
@@ -8,11 +8,79 @@
     <nav th:replace="~{fragments/navbar-fragments :: header-navbar}"></nav>
 </header>
 <section class="container">
-    <h1 th:text="${user.getFirstName}"></h1>
-    <h2>Profile Page</h2>
```
Comment (Dariusionut) [inline src/main/resources/templates/user.html:12]: th:text="${user.username + '  profile'}"

**Review by Dariusionut — CHANGES_REQUESTED** [review]

**Review by Dariusionut — APPROVED** [review]

**Review by aureliosss — COMMENTED** [review]
Great!

**Pull request merged**